### PR TITLE
StudentPeriodScreen

### PR DESCRIPTION
**Pull request by Ozamoram429-bit**

### CODE REVIEW

File: CareerPlan/CareerPlan.csproj [hunk 1]
```diff
@@ -15,4 +15,10 @@
     <PackageReference Include="Xamarin.Essentials" Version="1.6.1" />
   </ItemGroup>
 
+  <ItemGroup>
+    <EmbeddedResource Update="Views\StudentPeriod.xaml">
+      <Generator>MSBuild:UpdateDesignTimeXaml</Generator>
+    </EmbeddedResource>
+  </ItemGroup>
```
Comment (Gabuardi) [inline CareerPlan/CareerPlan.csproj:22]: This was added manually or auto-generated?

Comment (Ozamoram429-bit) [reply CareerPlan/CareerPlan.csproj:22]: Auto-generated!

Comment (Gabuardi) [reply CareerPlan/CareerPlan.csproj:22]: I think we can remove it, we don't need it

File: CareerPlan/Views/StudentPeriod.xaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="CareerPlan.Views.StudentPeriodPage">
+    <ContentPage.Content>
+        <StackLayout>
+            <Frame  Padding="24" CornerRadius="0">
+                <Label Text="Añadir Periodo Estudiantil" HorizontalTextAlignment="Center" TextColor="Black" FontSize="36" FontAttributes="Bold"/>
+            </Frame>
```
Comment (Gabuardi) [inline CareerPlan/Views/StudentPeriod.xaml:9]: We don't need this, just add the `Title` atribute to the `ContentPage` 

```xaml
<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
             x:Class="CareerPlan.Views.StudentPeriodPage"
             Title="Añadir Periodo Estudiantil">
```

**Review by Gabuardi — CHANGES_REQUESTED** [review]

**Review by Gabuardi — APPROVED** [review]

**Pull request merged**